src/app/containers/Article/index.jsx

```diff
@@ -34,11 +34,18 @@ class ArticleContainer extends Component {
   }
 
   render() {
+    const ID_REGEX = '(c[0-9]{8,15}o)$';
+
     const { data } = this.props;
     const { content, metadata, promo } = data;
+    const { id: aresArticleId } = metadata;
+
+    const idMatches = aresArticleId.match(ID_REGEX);
+    const id = idMatches ? idMatches[0] : '';
 
```

Comment:
@jtart I've addressed your comments.  

I'm not sure this part in this file is the ideal way of handling this. I've defaulted to setting the id to `''` here to prevent the whole component failing in case there isn't a match. I know this check should happen in the Data Validator part of the application, but I don't think we can guarantee that.
Any other comments on the PR would be welcome!

Reply:
Thanks. It would be simpler to just split the string at `:` and get the last index. We would very likely want fail the render if there is no ID, which we can do at a later time.

Reply:
Surely we would want to application to error if the id isn't valid. If we feel like this is something too important to only be handled in the validator I would suggest we have a validation step in this file so if something did sneak through the validator we would also catch it in the application code.